### ISSUE TITLE
Add current song to playlist

### ISSUE THREAD

**Issue by jugendhacker** [nvllsvm/Audinaut#69]
I just started using this nice app, but I'm missing a "add the current song to playlist" feature.... would it be difficult to add this?

**Comment by hialvaro** [comment]
Yes please, this is a very necessary feature for a better user experience
